blog/en/blog/2024/03/22/release-apache-apisix-3.9.0.md

```diff
@@ -0,0 +1,178 @@
+  -H "X-API-KEY: ${ADMIN_API_KEY}" \
+  -d '{
+    "id": "tls-client-ssl",
+    "sni": "example.com",
+    "cert": "'"${server_cert}"'",
+    "key": "'"${server_key}"'"
+  }'
+ ```
+
+For more information, see [PR #10989](https://github.com/apache/apisix/pull/10989), [PR #11010](https://github.com/apache/apisix/pull/11010), and [PR #11027](https://github.com/apache/apisix/pull/11027).
+
+### Add plugin reload endpoint to Control API
+
+The plugin reload endpoint, which previously existed in Admin API to hot reload all plugins, is now added to [Control API](https://apisix.apache.org/docs/apisix/next/control-api/#put-v1pluginsreload), such that the plugin reload can be called from control plane:
+
+```shell
+curl "http://127.0.0.1:9090/v1/plugins/reload" -X PUT
+```
+
+For backward compatibility, the plugin reload endpoint in Admin API is retained.
+
+For more information, see [PR #10905](https://github.com/apache/apisix/pull/10905).
+
+### Support setting session cookie lifetime in `openid-connect` plugin
+
+You can now configure the session cookie lifetime in the `openid-connect` plugin as such:
+
+```shell
+curl "http://127.0.0.1:9180/apisix/admin/routes" -X PUT -d '
+{
+  "id": "auth-with-oidc",
+  "uri":"/anything/*",
+  "plugins": {
+    "openid-connect": {
+      "client_id": "'"$OIDC_CLIENT_ID"'",
+      "client_secret": "'"$OIDC_CLIENT_SECRET"'",
+      "discovery": "'"$OIDC_DISCOVERY"'",
+      "session": {
+        "secret": "6S8IO+Pydgb33LIor8T9ClER0T/sglFAjClFeAF3RsY=",
+        "cookie" : {
+          "lifetime": 86400
+        }
+      },
+      "redirect_uri": "http://localhost:9080/anything/callback"
+    }
+  },
+  "upstream":{
+    "type":"roundrobin",
+    "nodes":{
+      "httpbin.org:80":1
+    }
+  }
+}'
+```
+
+This is to address the issue where the lifetime of the access token is longer than the session lifetime.
+
+For more information, see [PR #10919](https://github.com/apache/apisix/pull/10919).
+
+### Support setting expiration time for Prometheus metrics
+
+You can now set the expiration time in seconds after Prometheus metrics have become inactive to reduce resource consumption:
+
+```yaml title="config.yaml"
+plugin_attr:
+  prometheus:
+    expire: 0    # time in seconds. 0 means the metrics will not expire.
+```
+
+For more information, see [PR #10869](https://github.com/apache/apisix/pull/10869).
+
+### Support `redis` and `redis-cluster` policies in `limit-req` and `limit-conn` plugins
+
+Support `redis` and `redis-cluster` policies in `limit-req` and `limit-conn` plugins, so that rate limiting counters can be shared among APISIX instances.
+
+For more information, see [PR #10874](https://github.com/apache/apisix/pull/10874) and [PR #10866](https://github.com/apache/apisix/pull/10866).
+
+### Support OCSP Stapling
+
+Support OCSP stapling using the `ocsp-stapling` plugin.
+
+For more information, see [PR #10817](https://github.com/apache/apisix/pull/10817).
```

Comment:
Note that both `ocsp-stapling` and `http-dubbo` plugins do not have any doc as of now, so I have no more information to share here.

I've opened issues on apache/apisix repo for these missing docs.